Automattic/woocommerce-payments
# Adding Internal Logger

Fixes #7428 

#### Changes proposed in this Pull Request

- added `WCPay/Internal/Logger`
- modified `PaymentsServiceProvider` to provide the dependency
- Inject the Logger in `ExampleServiceWithDependencies.php` and `Level3Service.php`
- `WCPay/Logger` is modified to get `WCPay/Internal/Logger` from the container and use its methods

#### Testing instructions
- Check that existing logs are written
- Any new classes in src folder can use the `WCPay/Internal/Logger`


-------------------


**Post merge**
